### DISCUSSION TITLE
DIY Keyboard

### DISCUSSION THREAD

**Discussion by Sch-LikA** [fachat/MicroPET#8] [Ideas]
I also came accross this other option for a very nice keyboard, too bad I had already ordered the Texelec one :

- [Homepage](http://www.6502.org/users/sjgray/projects/mxkeyboards/index.html)
- [Github](https://github.com/sjgray/CBM-MX-Keyboards)

**Comment by fachat** [comment]
In fact I have both types of keyboards - texelec _and_ Steve's N-type keyboard, and can use both with the Micro-PETs in the C64-II case.

I have designed a new keyboard support plate ... a preview for it is here  https://www.tinkercad.com/things/a3phGKtNsUp

**Reply by Sch-LikA** [reply]
nice to know thanks !
